### ISSUE TITLE
OVA: systemd service admiral_startup runs twice

### ISSUE THREAD

**Issue by anchal-agrawal** [vmware/vic-product#442]
During OVA startup, `admiral_startup.service` seems to run twice. 

`journalctl` snippet (no reboot):

```
Jul 17 21:25:47 localhost.localdomain systemd[1]: Starting Admiral Startup Configuration...
Jul 17 21:25:47 localhost.localdomain bash[809]: Hostname is null, set it to IP
Jul 17 21:25:47 localhost.localdomain bash[809]: CA, Certificate or key file does not exist, will generate a self-signed certificate
Jul 17 21:25:47 localhost.localdomain bash[809]: Generating a 4096 bit RSA private key
Jul 17 21:25:48 localhost.localdomain bash[809]: ........................................................................................................................
.......................................................................................................................++
Jul 17 21:25:48 localhost.localdomain bash[809]: ......++
Jul 17 21:25:48 localhost.localdomain bash[809]: writing new private key to '/data/admiral/cert/ca.key'
Jul 17 21:25:48 localhost.localdomain bash[809]: -----
Jul 17 21:25:48 localhost.localdomain bash[809]: Generating a 4096 bit RSA private key
Jul 17 21:25:48 localhost.localdomain bash[809]: ........................................................................................................................
....................................................++
Jul 17 21:25:49 localhost.localdomain bash[809]: ................................................................................................++
Jul 17 21:25:49 localhost.localdomain bash[809]: writing new private key to '/data/admiral/cert/server.key'
Jul 17 21:25:49 localhost.localdomain bash[809]: -----
Jul 17 21:25:49 localhost.localdomain bash[809]: Add subjectAltName = IP: 10.160.148.251 to certificate
Jul 17 21:25:49 localhost.localdomain bash[809]: Signature ok
Jul 17 21:25:49 localhost.localdomain bash[809]: subject=/C=US/ST=California/L=Palo Alto/O=VMware/OU=Containers on vSphere/CN=10.160.148.251
Jul 17 21:25:49 localhost.localdomain bash[809]: Getting CA Private Key
Jul 17 21:25:49 localhost.localdomain bash[809]: unable to write 'random state'
Jul 17 21:25:49 localhost.localdomain bash[809]: Copy CA certificate to /data/admiral/ca_download
Jul 17 21:25:49 localhost.localdomain bash[809]: creating java keystore with self-signed CA
Jul 17 21:25:49 localhost.localdomain bash[809]: Certificate was added to keystore
Jul 17 21:25:49 localhost.localdomain bash[809]: [Storing /data/admiral/cert/trustedcertificates.jks]
Jul 17 21:25:49 localhost.localdomain bash[809]: Enable admiral startup
Jul 17 21:25:49 localhost.localdomain bash[809]: Created symlink from /etc/systemd/system/multi-user.target.wants/admiral_startup.service to /usr/lib/systemd/system/admi
ral_startup.service.
Jul 17 21:25:49 localhost.localdomain bash[809]: Created symlink from /etc/systemd/system/multi-user.target.wants/admiral.service to /usr/lib/systemd/system/admiral.serv
ice.
Jul 17 21:25:52 localhost.localdomain systemd[1]: Started Admiral Startup Configuration.

### second run below ###

Jul 17 21:25:54 localhost.localdomain systemd[1]: Starting Admiral Startup Configuration...
Jul 17 21:25:54 localhost.localdomain bash[1219]: Hostname is null, set it to IP
Jul 17 21:25:54 localhost.localdomain bash[1219]: Use the existing CA, certificate and key file
Jul 17 21:25:54 localhost.localdomain bash[1219]: Copy CA certificate to /data/admiral/ca_download
Jul 17 21:25:54 localhost.localdomain bash[1219]: Enable admiral startup
Jul 17 21:25:59 localhost.localdomain systemd[1]: Started Admiral Startup Configuration.
```

A similar service, `harbor_startup.service`, runs only once.

Here are the `systemd-analyze` graphs for both services: [systemd-analyze.zip](https://github.com/vmware/vic-product/files/1156414/systemd-analyze.zip)

**Comment by mdubya66** [comment]
@andrewtchin what's the impact of this on the customer?

**Comment by mdubya66** [comment]
making high, needs triage

**Comment by andrewtchin** [comment]
No impact, harbor_startup is designed to run each time the appliance boots anyway

**Comment by andrewtchin** [comment]
Closing as fixed by #1300
